include/git2/status.h

```diff
@@ -173,12 +173,17 @@ typedef enum {
  * The `pathspec` is an array of path patterns to match (using
  * fnmatch-style matching), or just an array of paths to match exactly if
  * `GIT_STATUS_OPT_DISABLE_PATHSPEC_MATCH` is specified in the flags.
+ *
+ * The `head_tree` is an optional tree to be used instaed of the head commit's
```

Comment:
`head_tree` is a bit misnamed as you're not trying to diff against a specific `head`. I have no real better idea right now, but at least remove the `head_` prefix. There's also a typo in "instead". Last, there is no need to specify that the tree is optional, as all options in our options arrays are typically optional.